### ISSUE TITLE
Crashes on launch

### ISSUE THREAD

**Issue by benrhughes** [kvakulo/Switcheroo#20]
After my latest reboot, Switcheroo is crashing on launch. I couldn't find a log file, but here's the crash from Event Viewer:

```
Application: switcheroo.exe
Framework Version: v4.0.30319
Description: The process was terminated due to an unhandled exception.
Exception Info: System.ComponentModel.Win32Exception
Stack:
   at ManagedWinapi.Hooks.Hook.StartHook()
   at Switcheroo.MainWindow.SetUpAltTabHook()
   at Switcheroo.MainWindow..ctor()
   at Switcheroo.Program.Main()
```
I tried deleting my user.config in the hope that it wouldn't try to re-hook Alt-Tab, but that didn't seem to help. I also tried installing/re-installing. Same thing.

Any ideas?

**Comment by ashutoshraina** [comment]
Hey @kvakulo ,
 
I tried the TC build and it works for me. The tagged releases were crashing as mentioned in the issue. Can we have the chocolatey packages updated along with an official release as well. 

 :+1:  for an awesome software, so happy to have this.

**Comment by kvakulo** [comment]
Thanks for the feedback @ashutoshraina!

I'll make an official release with the fix a little later today :)

/ Regin

**Comment by kvakulo** [comment]
Thanks @benrhughes and @ashutoshraina!

I have just released version 0.8.3 of Switcheroo containing this fix.

The chocolatey packages have been submitted and are waiting for approval.

/ Regin

**Comment by ashutoshraina** [comment]
awesome :+1:
